pkg/controllers/cloud/networkpolicy.go

```diff
@@ -772,13 +785,13 @@ func (a *addrSecurityGroup) notifyNetworkPolicyChange(r *NetworkPolicyReconciler
 	if !a.deletePending {
 		return
 	}
-	sgs, err := r.networkPolicyIndexer.ByIndex(networkPolicyIndexerByAddrGrp, a.id.Name)
+	nps, err := r.networkPolicyIndexer.ByIndex(networkPolicyIndexerByAddrGrp, a.id.Name)
 	if err != nil {
 		r.Log.Error(err, "Get networkPolicy indexer", a.id.Name, err, "indexKey", networkPolicyIndexerByAddrGrp)
```

Comment:
I know its legacy code, but is there any reason to proceed further if err occurs?

Reply:
No particular reason. We can try returning on error and see if it breaks anything. My guess is indexer error rarely happens and is probably never tested.